vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/SortingIT.java

```diff
@@ -79,9 +79,8 @@ public void setInitialSortOrder_changeOrderFromServer_sortIndicatorsUpdated() {
     public void setInitialSortOrder_sorterAriaLabels() {
         findElement(By.id("sort-by-age")).click();
         List<TestBenchElement> sorters = grid.$("vaadin-grid-sorter").all();
-        // The first sorter uses a ComponentRenderer so an aria-label can't be
-        // generated
-        Assert.assertEquals(false, sorters.get(0).hasAttribute("aria-label"));
+        Assert.assertEquals("Sort by Name",
+                sorters.get(0).getAttribute("aria-label"));
```

Comment:
The new connector method for rendering the header does fix: https://github.com/vaadin/web-components/issues/3470

Reply:
Thank you 🙇‍♂️